datastore/indexmanager.go

```diff
@@ -12,6 +12,8 @@ import (
 
 	"github.com/jeffpierce/cassabon/config"
 	"github.com/jeffpierce/cassabon/logging"
+
+	"github.com/otium/queue"
```

Comment:
Note to self: "github.com/otium/queue" needs to be vendored in https://github.com/jeffpierce/cassabon/pull/110